schema/mysql/schema.sql

```diff
@@ -588,6 +588,7 @@ CREATE TABLE downtime (
   is_in_effect enum('n', 'y') NOT NULL,
   start_time bigint unsigned DEFAULT NULL COMMENT 'Time when the host went into a problem state during the downtimes timeframe',
   end_time bigint unsigned DEFAULT NULL COMMENT 'Problem state assumed: scheduled_end_time if fixed, start_time + flexible_duration otherwise',
+  scheduled_by varchar(255) DEFAULT NULL,
```

Comment:
255 is probably not long enough. The value will be "host_name!service_name!downtime_name", so probably something like 3*255 + 2 is required.

Reply:
And there should be a `COMMENT` that explains that and what the column reflects.

Reply:
@lippserd Can you confirm that length?

Reply:
We have no limit on host and service names other than the implicit file name length limit for API-created objects, right?
I think this should be a text column just to be on the safe side. MySQL will not execute the statement if values need to be truncated and it is running in strict mode. Which in turn would stop Icinga DB.

Reply:
`host.name` and `service.name` are both `varchar(255)`. Assuming the same length for the names of ScheduledDowntimes, that yields the limit I suggested above. Haven't checked if Icinga 2 enforces these limits, but if it doesn't, maybe it should if we have database schemas with those limits?

Reply:
If the downtime name can't be longer than 255 chars 3*255+2 is fine. For host and service names we already have the 255 char length restriction in other tables. But yes, Icinga 2 should enforce those limits.

Reply:
I.e. Icinga 2 shall reject config of objects w/ names >255?